x-pack/plugins/fleet/common/http_authorization_header.ts

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import type { KibanaRequest } from '@kbn/core/server';
+
+// Extended version of x-pack/plugins/security/server/authentication/http_authentication/http_authorization_header.ts
+// to prevent bundle being required in security_solution
+// FIXME: Put this in a package
```

Comment:
For this FIXME comment do we have an issue for the fix? When do we expect to fix it?

Reply:
I removed this FIXME for now as the file is modified from the original copy of security_solution's code [`0187c5f` (#154665)](https://github.com/elastic/kibana/pull/154665/commits/0187c5f05136799f17c5b139d16206a44dbf7c55)